projects/burnr/src/themes/substrate/colors.ts

```diff
@@ -1,4 +1,4 @@
-export default {
+const colors = {
```

Comment:
Is this necessary (compared to the way it was before)?

Reply:
it's not necessary, bu it's a lot better for debugging reasons. There is now an eslint rule that raises a warning when creating anonymous default exports... And I really like that rule, b/c it really helps with debugging.